src/components/UserProfile/UserProfile.jsx

```diff
@@ -1,13 +1,30 @@
-import './UserProfile.css'
+import { useEffect } from "react";
+import "./UserProfile.css";
+import { useState } from "react";
+
+const UserProfile = () => {
+  const [users, setUsers] = useState([]);
+
+  useEffect(() => {
+    fetch("https://jsonplaceholder.typicode.com/users")
```

Comment:
Good job by using the fetch method, but we can improve it by adding a `.catch((error) => console.error(error))` and a `.finally(() => setIsLoading(false))` to change the state of the loading state

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/catch

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/finally